data/moves.ts

```diff
@@ -16040,8 +16001,6 @@ export const Moves: {[moveid: string]: MoveData} = {
 		sleepUsable: true,
 		onTry(source) {
 			if (source.status !== 'slp' && !source.hasAbility('comatose')) {
-				this.attrLastMove('[still]');
-				this.add('-fail', source);
 				return false;
 			}
 		},
```

Comment:
(Could this not be changed to something like `return source.status === 'slp' || source.hasAbility('comatose');`?)

Reply:
Yes, with Marty's new refactor of onTry() to have false automatically send 

``this.attrLastMove('[still]');
this.add('-fail', source); ``

on returning false, a few more of these can be further cleaned up. I'll see what remains.